index.html

```diff
@@ -0,0 +1,196 @@
+                            <img src="img/upload.png" alt="upload file">
+                            <label for="file">Upload Additional file</label>
+                            <input type="file" name="file" id="file">
+                        </div>
+                    </div>
+                    <p>File size of your documents should not exceed 10MB</p>
+                    <div>
+                        <input type="submit">
+                    </div>
+                    <div>
+                        <input type="radio" name="protectdata" id="protectdata">
+                        <label for="protectdata">I want to protect my data by signing an NDA</label> 
+                    </div>
+                </div>
+                <div>
+                    <div>
+                        <input type="checkbox" name="protect" id="protect" value="data">
+                        <label for="protect">I want to protect my data by signing an NDA</label>
+                    </div>
+                    <div>
+                        <input type="checkbox" name="terms" id="terms" value="conditions" checked>
+                        <label for="terms">I have read and accept the Terms and Conditions</label>
+                    </div>
+                </div>
+            </div>
+        </form>
+    </main>
+    <footer>
+        <div>
+            <h2>Career</h2>
+            <p>If you are interested in working at Logo send us your CV to <a
+                    href="careers@logo.com">careers@logo.com</a> and we can talk</p>
+            <div>
+                <a href="https://www.instagram.com/"><img src="img/instagram.png" alt="instagram"></a>
+                <a href="https://www.facebook.com/"><img src="img/facebook.png" alt="facebook"></a>
+                <a href="https://telegram.org/"><img src="img/telegram.png" alt="telegram"></a>
+            </div>
+        </div>
+        <div>
+            <h2>Offices</h2>
+            <address>United States of America<br>500 5th Avenue Suite 400, NY 10110</address>
+            <address>United Kingdom<br>High St, Bromley BR1 1DN</address>
+            <address>France<br>80 avenue des Terroirs de France, Paris</address>
+        </div>
+        <div>
+            <h2>For Quick Inquiries</h2>
+            <ul>
+                <li>
+                    <img src="img/great_britain.png" alt="great britain">
+                    <div>
+                        <a href="tel:+44 7911 123456">+44 7911 123456</a>
+                    </div>
+                </li>
+                <li>
+                    <img src="img/usa.png" alt="usa">
+                    <div>
+                        <a href="tel:+1-212-456-7890">+1-212-456-7890</a>
+                    </div>
+                </li>
+            </ul>
+            <form>
+                <label for="email image">Join our newsletter!</label>
+                <div>
+                    <input type="email" name="email" id="email image" placeholder="email">

```

Comment:
Name of the id:

- id must be **unique** in the document
- must not contain any **space** characters
- must contain at least one character

Fix please and keep it in mind for future